package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "wechaty-puppet-puppeteer",
-  "version": "0.17.29",
+  "version": "0.17.30",
```

Comment:
If you want to publish a new version to the `@latest` tag, you need to bump the minor version number, instead of the patch number.

I'd like to suggest that we can upgrade to `0.18.0` for your convenience.

Reply:
thanks for instruction. Working on that